<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="appledeMacBook-Pro-72.local" name="testReport" tests="8" failures="6" timestamp="14 Aug 2019 10:08:06 GMT" time="0.018" errors="0">
  <testcase name="runnerT1" time="0.002" classname="finup.feather.controller.TestReport"/>
  <testcase name="runnerT2" time="0.0" classname="finup.feather.controller.TestReport"/>
  <testcase name="runnerF2" time="0.002" classname="finup.feather.controller.TestReport">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at finup.feather.controller.TestReport.runnerF2(TestReport.java:24)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- runnerF2 -->
  <testcase name="runnerF1" time="0.002" classname="finup.feather.controller.TestReport">
    <failure type="java.lang.AssertionError" message="FFFF1111message expected [bb] but found [aa]">
      <![CDATA[java.lang.AssertionError: FFFF1111message expected [bb] but found [aa]
	at finup.feather.controller.TestReport.runnerF1(TestReport.java:19)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- runnerF1 -->
  <testcase name="runnerF1" time="0.0" classname="finup.feather.controller.TestReport">
    <failure type="java.lang.AssertionError" message="FFFF1111message expected [bb] but found [aa]">
      <![CDATA[java.lang.AssertionError: FFFF1111message expected [bb] but found [aa]
	at finup.feather.controller.TestReport.runnerF1(TestReport.java:19)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- runnerF1 -->
  <testcase name="runnerF2" time="0.0" classname="finup.feather.controller.TestReport">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at finup.feather.controller.TestReport.runnerF2(TestReport.java:24)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- runnerF2 -->
  <testcase name="runnerF2" time="0.0" classname="finup.feather.controller.TestReport">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at finup.feather.controller.TestReport.runnerF2(TestReport.java:24)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- runnerF2 -->
  <testcase name="runnerF1" time="0.0" classname="finup.feather.controller.TestReport">
    <failure type="java.lang.AssertionError" message="FFFF1111message expected [bb] but found [aa]">
      <![CDATA[java.lang.AssertionError: FFFF1111message expected [bb] but found [aa]
	at finup.feather.controller.TestReport.runnerF1(TestReport.java:19)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- runnerF1 -->
</testsuite> <!-- testReport -->
